modules/installation-user-infra-machines-advanced-customizing-iso-or-pxe.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+
+[id="installation-user-infra-machines-advanced-customizing-iso-or-pxe_{context}"]
+= Customizing a live {op-system} ISO or PXE install
+You can use the live ISO image or PXE environment to install {op-system} by injecting an Ignition config file directly into the image. This creates a customized image which you can then use to boot a live system that is provisioned with the given Ignition config.
+
+For an ISO image, the mechanism to do this is the `coreos-installer iso customize` subcommand which modifies the `.iso` file with your configuration. Similarly, the mechanism for a PXE environment is the `coreos-installer pxe customize` subcommand which creates a new initramfs file that combines your configuration with the PXE image.
+
+In addition to embedding Ignition configs directly, the `customize` subcommand is a general-purpose tool that can take multiple other arguments and create Ignition configs from them to embed in the live system. Some common tasks include:
```

Comment:
For simple cases that have user-friendly `customize` arguments, the user doesn't really need to understand the flow of data from live to destination Ignition configs.  By casually mentioning live configs in this section without explaining, I fear we're introducing too much confusion.  Let's generally try to focus more on the user's goals and deemphasize the "live system" aspect of the implementation.

Reply:
Thanks, I've updated this though please let me know if the level of detail is still off: my goal for this module is to introduce `customize` as a general purpose tool and set the user up a little bit for the examples they'll encounter in later sections.

Reply:
I think this looks good now. :+1: